src/gateway_gatt_application.erl

```diff
@@ -0,0 +1,20 @@
+-module(gateway_gatt_application).
+
+-behavior(gatt_application).
```

Comment:
So this application behaves as a gatt_application which behaves as an ebus_object.  Quite a bit to unpack here.